### PR TITLE
feat(RemediationsButton): RHICOMPL-2052 - Pass OUIA id for button

### PR DESCRIPTION
**Pull request by bastilian**
This adds an OUIA id for the compliance button to allow target it better in tests.

Note: This requires https://github.com/RedHatInsights/frontend-components/pull/1241

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by skateman** [comment]
ou ya :+1:

File: src/PresentationalComponents/ComplianceRemediationButton/ComplianceRemediationButton.js [hunk 1]
```diff
@@ -108,6 +108,9 @@ class ComplianceRemediationButton extends React.Component {
             addNotification(result.getNotification())
           }
           dataProvider={this.dataProvider}
+          buttonProps={{
+            ouiaId: 'RemediateButton',
```
Comment (akofink) [inline src/PresentationalComponents/ComplianceRemediationButton/ComplianceRemediationButton.js:112]: @LightOfHeaven1994 do you think we need to have different IDs for the remediate button for systems (report details page) vs. rules (system details page)?

Comment (skateman) [reply src/PresentationalComponents/ComplianceRemediationButton/ComplianceRemediationButton.js:112]: I'm not @LightOfHeaven1994 but I guess it's good to know what path our users are taking to reach the button :thinking:

Comment (akofink) [reply src/PresentationalComponents/ComplianceRemediationButton/ComplianceRemediationButton.js:112]: I'm quite sure they search ouid from within a specific page in IQE. Pendo also is able to determine the page, so I think we're probably OK having the same ID for the button in both places.

**Review by akofink — APPROVED** [review]
Thanks, @bastilian!

File: src/PresentationalComponents/ComplianceRemediationButton/ComplianceRemediationButton.js (hunk 1 above)
Comment (LightOfHeaven1994) [reply src/PresentationalComponents/ComplianceRemediationButton/ComplianceRemediationButton.js:112]: @akofink I'm okay with having the same name for both pages, 'RemediateButton' works for us, thanks :)

**Pull request merged**